rest_api/controller/search.py

```diff
@@ -97,15 +98,56 @@
 FINDERS = {1: Finder(reader=reader, retriever=retriever)}
 
 
+# TODO: move to correct location or make staticmethod
```

Comment:
Since it's only required for generating a question request, we could move to the `Question` class?